spot_rl_experiments/spot_rl/envs/place_env.py

```diff
@@ -2,69 +2,159 @@
 # This source code is licensed under the MIT license found in the
 # LICENSE file in the root directory of this source tree.
 
+import os
+import sys
+import time
 
 import magnum as mn
 import numpy as np
 from spot_rl.envs.base_env import SpotBaseEnv
 from spot_rl.real_policy import PlacePolicy
+from spot_rl.utils.generate_place_goal import get_global_place_target
 from spot_rl.utils.utils import (
     construct_config,
     get_default_parser,
-    place_target_from_waypoints,
+    get_waypoint_yaml,
+    place_target_from_waypoint,
 )
 from spot_wrapper.spot import Spot
 
+DOCK_ID = int(os.environ.get("SPOT_DOCK_ID", 520))
 
-def main(spot):
+
+def parse_arguments(args=sys.argv[1:]):
     parser = get_default_parser()
-    parser.add_argument("-p", "--place_target")
-    parser.add_argument("-w", "--waypoint")
-    parser.add_argument("-l", "--target_is_local", action="store_true")
-    args = parser.parse_args()
-    config = construct_config(opts=args.opts)
+    parser.add_argument(
+        "-p",
+        "--place_target",
+        help="input:float,float,float -> place target x,y,z in meters",
```

Comment:
Hi Kavit, thanks for this! Could you please do `type=list[float]`? Same for the following arguments.